lib/event_bus/services/store.ex

```diff
@@ -37,7 +37,7 @@ defmodule EventBus.Service.Store do
   @spec fetch(event_shadow()) :: event() | nil
   def fetch({topic, id}) do
     case Ets.lookup(table_name(topic), id) do
-      [{_, %Event{} = event}] -> event
+      [{_, data}] -> data
```

Comment:
What is the idea behind removing the filter, are we expecting any other data to flow in this context?

Reply:
Sorry, it should be left as is. Error on my part. Going to revert

Reply:
Oh right, thanks for mentioning. Going to revert that.